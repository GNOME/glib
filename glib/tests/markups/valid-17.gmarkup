<foo tab="	" />
<bar tab_character_reference="&#9;" />
