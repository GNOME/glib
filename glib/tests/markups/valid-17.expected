ELEMENT 'foo'
tab=" "
END 'foo'
ELEMENT 'bar'
tab_character_reference="	"
END 'bar'
